### ISSUE TITLE
rule "newline-after-var" might add option to allow comment line instead of newline

### ISSUE THREAD

**Issue by doberkofler** [eslint/eslint#2246]
Based on my experience (#2237) with this rule, I would recommend to add an option that also allows using a comment line instead of an empty newline.

```javascript
/*eslint newline-after-var:[2, "always"]*/

var foo = "bar";
//  This will not cause a warning
doSomethingWith(foo)
```

**Comment by nzakas** [comment]
I'm not sure about this suggestion. The rule is specifically called "newline after var". Why would a comment be allowed?

**Comment by doberkofler** [comment]
In my opinion this rule should optically separate the variable declarations from the rest of a function and when testing the regression i liked the idea that not only an empty line (newline) but also a comment line might be used as separator.

**Comment by nzakas** [comment]
I'm sorry, I'm having a lot of trouble rationalizing this. This just doesn't make sense to me in the context of this rule. I know you like it, but I don't see how such functionality is appropriate given the purpose of the rule.
